ISSUE_TEMPLATE/compatibility_template.md

```diff
@@ -0,0 +1,36 @@
+# Incompatibility Report
+- Name of the game with compatibility issues:
+- Steam AppID of the game: 
+
+## System Information
+- Driver version<!-- e.g. Mesa 18.2.0-devel -->:
+- Kernel version<!-- e.g. 4.17 -->:
+- LLVM version [optional]<!-- e.g. 7.0.0 -->:
+- Graphics processor<!-- e.g. RX 580 -->:
+- Link to full system information report as [Gist](https://gist.github.com/):
+- Proton version:
+
+## I confirm:
+- [ ] that I haven't found an incompatibility report for this game with the
+      provided symptoms.
+- [ ] that I have checked whether there are updates for my system available.
+
```

Comment:
It would also be good to have:
Please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the issue report.